verify/cockpit-verify

```diff
@@ -28,7 +25,12 @@ while true; do
     git -C cockpit fetch origin master
     git -C cockpit reset --hard FETCH_HEAD
     cockpit/test/vm-download --prune
+
+    if ! git diff -C cockpit --exit-code origin/master package.json; then
```

Comment:
This will leave an updated install for subsequent runs with branches that don't deviate from master's package.json, but it's probably ok for now.